docs/source/usage.rst

```diff
@@ -0,0 +1,13 @@
+Usage
+=====
+
+.. _installation:
+
+Installation
+------------
+
+To use QuCoNot, first install it and its dependencies from PyPI with:
+
+.. code-block:: console
+
+   (.venv) $ pip install quconot
```

Comment:
This does not work yet? perhaps for the moment we should make the easy steps (clone/download + requirements install)

Reply:
Opened issue #24 for this.
All files will be updated anyway